### DISCUSSION TITLE
uuid in uuid[] does not exist

### DISCUSSION THREAD

**Discussion by abeldebruijn** [supabase/supabase#4911] [Questions] [answered]
I want to write a policy where I check if a column's `group_author` (uuid) is included in an array of uuid's. 

<img width="1007" alt="Screenshot 2022-01-10 at 14 31 54" src="https://user-images.githubusercontent.com/25432790/148775073-7911cc99-e96e-4489-b792-3e1bafb829c4.png">

I essentially want to know if it is possible to write a policy where `'123' in array['123', '456']` is true?

**Comment by GaryAustin1** [comment, marked as answer]
I did a similar thing with: using ((room_key)::text = ANY ((my_rooms())::text[]));  You may just need the = ANY versus in. Note my keys are text but should work for uuid also.
